### ISSUE TITLE
obtain quote for DNA methylation analysis

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1732]
For Pacific Cod 40 samples... 

Looking for WGBS and/or RRBS...

V interested in option for Revio Sequencing option https://pacbio.gs.washington.edu/

**Comment by kubu4** [comment]
What starting material (i.e. DNA or tissue? If tissue, what type of tissue?)?

**Comment by kubu4** [comment]
Eh, doesn't matter. They only accept DNA as starting material.

**Comment by kubu4** [comment]
How much coverage are we wanting? Minimum 10x?

**Comment by sr320** [comment]
Will go on their recommendations... but shoot for 20x

**Comment by sr320** [comment]
@mgavery -what is recommendation for doing fish methylation?

**Comment by mgavery** [comment]
WGBS over RRBS if you have the ability (i.e. $).  RRBS covers such a tiny amount of genome.  Do you have a good genome?

**Comment by kubu4** [comment]
If doing RRBS, would we have an inkling of how much the genome would be reduced (i.e. number of bases we'd expect to recover?)? Since most of the sequencing quotes are based on desired coverage, this would be a good number to have.

**Comment by mgavery** [comment]
I did this for Steelhead a million years ago. Kind of an obnoxious exercise, but made fragments based on MspI (CCGG) and then filtered to those in the range for RRBS libraries (depends on library prep) to see how many bases could potentially be sequenced

**Comment by kubu4** [comment]
Bioinformatically?

**Comment by mgavery** [comment]
Yes, bioinformatically. There was an old software program we had that would make the fragments based on enzyme cutsites. Then I just pulled out in silico fragments between 100-300bp (or something close to that, I forget how big the fragments in the library were) and based it off the estimate of total bases from those fragments.

**Comment by sr320** [comment]
Let's do WGBS not RRBD

**Comment by kubu4** [comment]
UW PacBio Revio:

TL;DR: 40 Cod samples -> 7 SMRT Cells, 6 samples per SMRT Cell = ~$29


Wall of text response from UW PacBio:

>Starting with your WGS projects (Cod and Oyster.) I want to make sure I clarify before we begin that the methylation signatures called onboard by Revio is currently limited to mCpG. At this time, cytosine methylation in other base contexts is not called. Typically this is a non-issue for human sequencing but depending on the organism of interest, this may be a limitation of the method. There may be post-acquisition analyses you could run (e.g. fibertools calls m6A modifications, and bacterial motifs can be called for 4mC and 6mA.) We can retain the raw kinetics information that could be used for post-acquisition analysis but this typically results in file sizes ~10X larger than files without the embedded kinetics.

> Typically we average 90 Gbp/Revio SMRT Cell, but based on past experience with marine invertebrates, I'd suggest revising that number down and I'll do my math with an 80 Gbp/SC expectation. In this case I'd plan to pool 6 cod samples or 7 oysters per Revio SMRT Cell (~23X coverage expected per sample). To save money and processing, I'd recommend pooling before size selection (3- or 4-plex per size selection pool) -- this way if topoff is necessary it limits the extraneous sequence that will be generated. We would need about 3-5 µg of high-quality extracted gDNA (peak size > 50kbp) per sample in order to begin. Unfortunately, we do not offer extraction services at this time. These are significant projects and we will have to work out a schedule for sample and data delivery. I'd generally say we should be able to process about 2 SMRT Cells worth of samples in about 3-4 weeks with proper scheduling and planning. 40 Cod samples -> 7 SMRT Cells, 6 samples per SMRT Cell = ~$29k; 60 oyster samples -> 8 SMRT Cells, 7 samples per SMRT Cell = ~$36k.

**Comment by kubu4** [comment]
Zymo quote requested.

**Comment by kubu4** [comment]
Azenta quote requested.

**Comment by kubu4** [comment]
Macrogen quote requested.

**Comment by kubu4** [comment]
Macrogen quote: `$14,400` ($360/sample)

**Comment by sr320** [comment]
Determine cost for inhouse library prep using https://www.neb.com/en-us/products/e7120-nebnext-enzymatic-methyl-seq-kit#Product%20Information

and subsequent sequence costs

**Comment by kubu4** [comment]
I've submitted a support question to NEB regarding multiplexing (not clear how to multiplex >4 samples (24 rxn kit) or >8 samples (96 rxn kit). Will get sequencing quote(s) after I have a better understanding of how many sequencing lanes we might need.

**Comment by kubu4** [comment]
Azenta quote: `$27,210` ($680/sample):

[30-945334233_R3.pdf](https://github.com/RobertsLab/resources/files/13170006/30-945334233_R3.pdf)

**Comment by kubu4** [comment]
Requested quote for just sequencing (assumes we prepare libraries) to UW Northwest Genomics Center.

**Comment by sr320** [comment]
@kubu4 any word from UW Northwest Genomics Center?

**Comment by sr320** [comment]
> I've submitted a support question to NEB regarding multiplexing (not clear how to multiplex >4 samples (24 rxn kit) or >8 samples (96 rxn kit). Will get sequencing quote(s) after I have a better understanding of how many sequencing lanes we might need.

anything regarding this?

**Comment by sr320** [comment]
> For paired-end 100bp reads, the cost is $11,660
for paired-end 150bp reads, the cost is $12,356.

to clarify.. this is for us to make NEB libraries and UW Northwest Genomics Center to sequence?

**Comment by kubu4** [comment]
Correct.

**Comment by sr320** [comment]
Lets go with Macrogen 
https://github.com/RobertsLab/resources/issues/1732#issuecomment-1776059646 - 
@kubu4 do we need an updated quote?

**Comment by kubu4** [comment]
Updated Macrogen quote (pricing actually _decreased_!).

I'll get PO request process initiated.

>Hi Sam, 

>
>It's good to hear from you and thank you for reaching out. Here is our latest pricing on the WGBS. Please see the below and let me know if you have any questions. 

>
>If you're good with the details below and ready to move forward, please visit our website ([order.psomagen.com](http://order.psomagen.com/)) and create an account!

>
>[WGBS Project Design]
Application – WGBS
Species – Fish (Gadus macrocephalus; Pacific cod)
Starting material – extracted gDNA
Sample size – 40 samples
Library prep kit – Accel-NGS Methyl-seq DNA library
Platform – NovaSeqX+ 150PE 
Output # - 12Gb raw throughput/sample (targeting 20x coverage)
Deliverable – fastq raw data
Data delivery method – AWS or HDD delivery
Timeline – 4~6 weeks (after initial sample QC)  
Price - sample QC + library prep + library QC + sequencing = $310/sample

>Best,
Hobin

**Comment by kubu4** [comment]
Adding the official quote here:

[Quotation_ANQ00004072_20240122_WGBS_40.pdf](https://github.com/RobertsLab/resources/files/14027937/Quotation_ANQ00004072_20240122_WGBS_40.pdf)

**Comment by laurahspencer** [comment]
For this sequencing run we plan to include DNA from a few different Pacific cod tissue types (blood, fin, jawbone). The blood and fin DNA is coming from our lab. The jawbone DNA probably needs to be sent from a different lab that specializes in bone DNA extraction. Will that be okay?  

And just to triple check- we're sending DNA and they are doing the bisulfite treatment etc.? 

We also plan to send a few samples from the Pacific sleeper shark, which probably has a very large genome (perhaps as large as 12Gb). Since this quote is for 12 GB/sample, this could result in low coverage.  Can the facility increase data for a few sleeper shark samples?

**Comment by sr320** [comment]
See
https://github.com/RobertsLab/resources/issues/1884#issuecomment-2223673100

**Comment by kubu4** [comment]
Need new quote, as [previous quote](https://github.com/RobertsLab/resources/issues/1732#issuecomment-1906587391) expired in April.

Will send email today.
